ASP_tests.py

```diff
@@ -267,9 +267,25 @@ def test_extract_sn_from_parquet_file_and_write_to_csv():
                                                   output_path,
                                                   mag_limits=[20, 21])
     sn_ids = pd.read_csv(output_path, header=None).values.flatten()
-    test_sn_ids = pd.read_csv(pathlib.Path(__file__).parent/"tests/testdata/test_snids.csv", header=None).values.flatten()
-    assert np.array_equal(sn_ids, test_sn_ids), "The SNIDs do not match the test example"
-    
+    test_sn_ids = pd.read_csv(pathlib.Path(__file__).parent
+                              / "tests/testdata/test_snids.csv",
+                              header=None).values.flatten()
+    np.testing.assert_array_equal(sn_ids, test_sn_ids), \
+        "The SNIDs do not match the test example"
+
+
+def test_extract_star_from_parquet_file_and_write_to_csv():
+    output_path = pathlib.Path(__file__).parent/"tests/testdata/star_ids.csv"
```

Comment:
Race condition.  Write output to temporary directory guaranteed not to conflict.